.github/workflows/docs.yml

```diff
@@ -11,5 +11,5 @@ jobs:
       with:
         docs-folder: "doc/"
         repo-token: "${{ secrets.GITHUB_TOKEN }}"
-        pre-build-command: "apt install -y graphviz && pip3 install -r doc/requirements.txt && pip3 install . && pip3 install -r qchem/requirements.txt && pip3 install ./qchem"
+        pre-build-command: "apt install -y graphviz && pip3 install -r doc/requirements.txt && pip3 install . pip3 install openfermionpyscf"
```

Comment:
Does `openfermionpyscf` depend on `pyscf`?

Reply:
Yes, it actually installs both `openfermion` and `pyscf`.

Reply:
I had to add `openfermionpyscf` and `pyscf` to `requirements-ci`, otherwise Codecov complains about the tests that were skipped by `openfermionpyscf = pytest.importorskip("openfermionpyscf")`. 

This means that users and developers will not forced to install these packages and they still get all their other tests passed and some of the qchem tests skipped. But CI will run all of the tests for each PR. 

Also, had to install pyscf from source because of an installation error discussed [here](https://github.com/pyscf/pyscf/issues/1093).